src/components/apps/launch/AppInfo.js

```diff
@@ -105,25 +111,94 @@ const UnavailableMsg = ({ app, hasDeprecatedParams, baseId }) => {
 };
 const AppInfo = React.forwardRef((props, ref) => {
     const { app, baseId, hasDeprecatedParams, loading, loadingError } = props;
+    const { t } = useTranslation("common");
+    const { t: i18nApps } = useTranslation("apps");
```

Comment:
I actually prefer declaring separate functions ever since I learned that array syntax needs explicitly used for one simple reason - the prefix syntax seems specific to this framework so it might stump a new dev / community dev while the other syntax is just plain js 🤷‍♂️ . But I do agree this syntax is more terse.